### DISCUSSION TITLE
Can someone explain the Negative TimeToGo numbers to me

### DISCUSSION THREAD

**Discussion by optio50** [Louisvdw/dbus-serialbattery#758] [General]
While charging and battery is at some percentage (say 75%) why does it display both positive and negative numbers.
Negative below the current state of charge and positive above the current state of charge?

Conversely it happens when discharging.

I would expect only positive numbers in the direction of travel whether discharging or charging.
The time to get to a certain point of charge is a positive number either way.

**Comment by mr-manuel** [comment]
You probably mean the Time to SoC? This was fixed and is available in the nightly dev build.

**Reply by optio50** [reply]
Yes, That's what I meant. Thanks
